spec.md

```diff
@@ -700,94 +697,85 @@ The response format is as follows:
         ]
     }
 
-### Listing Repositories
 
-Images are stored in collections, known as a _repository_, which is keyed by a `name`, as seen throughout the API specification.
-A registry instance MAY contain several repositories.
-The list of available repositories is made available through the _catalog_.
+### Listing Image Tags
```

Comment:
?? I think you wanted to pick up the interesting description of how pagination works and merge that down into the pagination description in image tags.  Not move some portions of the description of image tags into the description of listing repositories. See references to repositories in the pagination examples below.